packages/dev/docs/pages/releases/2025-03-05.mdx

```diff
@@ -0,0 +1,133 @@
+{/* Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero, Image} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+import datepickerImage from 'url:../assets/datepicker-rtl.png';
+
+---
+description: 
```

Comment:
Just a heads up, but this will need to be updated before we merge. Perhaps 
```
Buckle in because this is a big one! In this React Aria release, we are excited to announce that Toast, Tree, and Virtualizer are now in GA, along with improvements to Autocomplete, support for custom Menu, Popover, and Tooltip triggers, and more! For React Spectrum, Toast and TreeView are now in GA, alongside support for tooltips in TagGroup and Tabs, and improved localization support in our Datepicker. 
```
?

Reply:
How about this? Just wanted to add something about usePress and separate Toast from Tree and Virtualizer since it's not going to GA

> Buckle in because this is a big one! In this React Aria release, we are excited to announce that Tree and Virtualizer are now in GA, while Toast is now in available as an alpha. We've also made improvements to Autocomplete, added support for custom Menu, Popover, and Tooltip triggers, completed a major refactor of usePress, and more! For React Spectrum, Toast and TreeView are now in GA, with added support for tooltips in TagGroup and Tabs, and improved localization support in DatePicker.

Reply:
I'm fine with that